source/source.go

```diff
@@ -48,11 +47,11 @@ const (
 	// The annotation used for specifying the type of endpoints to use for headless services
 	endpointsTypeAnnotationKey = "external-dns.alpha.kubernetes.io/endpoints-type"
 	// The annotation used for defining the desired ingress/service target
-	targetAnnotationKey = "external-dns.alpha.kubernetes.io/target"
+	targetAnnotationKey = annotations.TargetAnnotationKey
```

Comment:
we should get rid of the "Annotation" part in the name of our annotations:
`annotations.TargetAnnotationKey`-> `annotations.TargetKey`

Reply:
done